IO Section (#173)
ankitects/anki-manual: src/getting-started.md

  deletion (e.g., “Man landed on the moon in \[…​\]” → “Man landed on the
  moon in 1969”). More information is available in the [cloze deletion](editing.md#cloze-deletion) section.

- **Image Occlusion**\

>   A special case of Cloze Note Type, that works with images instead of text.

Reviewer: ```suggestion
  Like the cloze notetype, but it works with images instead of text,
```

>   This Note type allows you to create cards that hide some parts of an image,

Reviewer: ```suggestion
```
